packages/vue/src/components/organisms/SfStoreLocator/SfStoreLocator.vue

```diff
@@ -178,7 +171,16 @@ export default {
       },
     },
   },
-  mounted() {
+  async mounted() {
+    // Fix lack of marker icons
+    const { icon } = await import('leaflet');
```

Comment:
I think you shouldn't change the capitalized **Icon** to lowercased **icon**. 

It was like class where were overridden default options after import.

Reply:
Good catch